oak/server/base_runtime.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OAK_SERVER_BASE_RUNTIME_H_
+#define OAK_SERVER_BASE_RUNTIME_H_
+
+namespace oak {
+
+class BaseRuntime {
```

Comment:
Not sure what this is for, is it in order to have a weak reference from the Wasm node to the runtime? Could you add a comment?

Reply:
Added a comment (there was some explanation, but it was buried in the commit message which isn't very helpful).